x-pack/solutions/security/plugins/elastic_assistant/server/ai_assistant_data_clients/knowledge_base/index.ts

```diff
@@ -327,6 +335,40 @@ export class AIAssistantKnowledgeBaseDataClient extends AIAssistantDataClient {
         }
       }
 
+      // Load Defend Insights docs if not already loaded
+      this.options.logger.debug(`Checking if Defend Insights docs have been loaded...`);
+      const defendInsightsDocsLoaded = await this.isDefendInsightsDocsLoaded();
+      if (!defendInsightsDocsLoaded) {
+        // Delete any existing Defend Insights content
+        const defendInsightsDocs = await (
+          await this.options.elasticsearchClientPromise
+        ).deleteByQuery({
+          index: this.indexTemplateAndPattern.alias,
+          query: {
+            bool: {
+              must: [
+                {
+                  bool: {
+                    must: [{ prefix: { kb_resource: DEFEND_INSIGHTS_RESOURCES } }],
+                  },
+                },
+              ],
+            },
+          },
+        });
+
+        if (defendInsightsDocs?.total) {
+          this.options.logger.info(
+            `Removed ${defendInsightsDocs?.total} Defend Insights knowledge base docs from knowledge base data stream: ${this.indexTemplateAndPattern.alias}.`
+          );
+        }
+
+        this.options.logger.debug(`Loading Defend Insights KB docs...`);
+        void loadDefendInsights(this, this.options.logger);
```

Comment:
I think we'll need to adjust the `then()` above:

```
void loadSecurityLabs(this, this.options.logger)?.then(() => {
  this.options.setIsKBSetupInProgress(this.spaceId, false);
});
```

else it's going to think setup is complete after the labs content loads but maybe not the new defend insights? Actually, might not be an issue since the there is no `await` and the labs docs will take much longer to load? Will need to test, but I think we'll want to re-work this a little.